cloudflare/resource_cloudflare_split_tunnel_exclude.go

```diff
@@ -0,0 +1,127 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudflareSplitTunnelExclude() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceCloudflareSplitTunnelsExcludeRead,
+		Update: resourceCloudflareSplitTunnelsExcludeUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"tunnels": {
+				Computed: true,
```

Comment:
do we really want this to be marked as `Computed`? i'd expect this would probably need to be an `Required` field otherwise we don't really have a resource to manage.

Reply:
Also these docs (https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers) is what gave me the idea that we needed `Computed: true`. I thought because the API doesn't explicitly return a field called `tunnels` that it should have `computed` on it.